backend/frontend/static/frontend/user_board.js

```diff
@@ -21,3 +21,54 @@ function moveCardToCorrectColumn(cardId, column) {
     destinationColumn.insertBefore(card, destinationColumn.firstChild);
   }
 }
+
+function showCardAlert({
+  cardId,
+  message,
+  alertType = "info",
+  persist = true,
+}) {
+  const alertThemes = {
+    success: "border-green-300 bg-green-100 text-green-800",
+    error: "border-red-300 bg-red-100 text-red-800",
+    warning: "border-yellow-300 bg-yellow-100 text-yellow-800",
+    info: "border-blue-300 bg-blue-100 text-blue-800",
+  };
+
+  const iconThemes = {
+    success: "fa-check-circle",
+    error: "fa-times-circle",
+    warning: "fa-exclamation-circle",
+    info: "fa-info-circle",
+  };
+
+  const card = document.getElementById(`card_${cardId}`);
+  const alert = document.createElement("div");
+  const cardButtons = card.querySelectorAll("button");
+
+  cardButtons.forEach((cardButton) => (cardButton.disabled = true));
```

Comment:
Why disable and re-enable the buttons? What if the user wants to immediately click the details button? They would have to wait.